Add 1 on 1 video calling/client.js

```diff
@@ -0,0 +1,159 @@
+    } else {
+        rendererRemote.dispose();
+    }
+  });
+  if (remoteVideoStream.isAvailable) {
+    remoteVideoView();
+  }
+}
+
+function subscribeToRemoteParticipant(p) {
+  p.videoStreams.forEach(v => {
+    handleVideoStreams(v);
+  });
+  p.on('videoStreamsUpdated', e => {
+    e.added.forEach(v => {
+      handleVideoStreams(v);
+    })
+  });
+}
+
+
+async function init() {
+  const callClient = new CallClient();
+  const tokenCredential = new AzureCommunicationTokenCredential("<USER ACCESS TOKEN>");
+  callAgent = await callClient.createCallAgent(tokenCredential, { displayName: 'optional ACS user name' });
+      
+  deviceManager = await callClient.getDeviceManager();
+  callButton.disabled = false;
+
+  callAgent.on('incomingCall', async e => {
+    const videoDevices = await deviceManager.getCameras();
+    const videoDeviceInfo = videoDevices[0];
+    localVideoStream = new LocalVideoStream(videoDeviceInfo);
+    localVideoView();
+
+    stopVideoButton.disabled = false;
+    callButton.disabled = true;
+    hangUpButton.disabled = false;
+
+    const addedCall = await e.incomingCall.accept({videoOptions: {localVideoStreams:[localVideoStream]}});
+    call = addedCall;
+
+    addedCall.remoteParticipants.forEach( p => {
+      subscribeToRemoteParticipant(p);
+    })
+
+    addedCall.on('remoteParticipantsUpdated', e => {
+      e.added.forEach( p => {
+        subscribeToRemoteParticipant(p);
+      })
+    });        
+  });
+
+  callAgent.on('callsUpdated', e => {
+    e.removed.forEach(removedCall => {
+      // dispose of video renderers
+      rendererLocal.dispose();
+      rendererRemote.dispose();
+      // toggle button states
+      hangUpButton.disabled = true;
+      callButton.disabled = false;
+      stopVideoButton.disabled = true;
+    })
+  })
+}
+init();
+
+async function localVideoView() {
+  rendererLocal = new Renderer(localVideoStream);
+  const view = await rendererLocal.createView();
+  document.getElementById("myVideo").appendChild(view.target);
+}
+
+async function remoteVideoView() {
+  rendererRemote = new Renderer(remoteVideoStream);
+  const view = await rendererRemote.createView();
+  document.getElementById("remoteVideo").appendChild(view.target);
+}
+
+callButton.addEventListener("click", async () => {
+  const videoDevices = await deviceManager.getCameras();
+  const videoDeviceInfo = videoDevices[0];
+  localVideoStream = new LocalVideoStream(videoDeviceInfo);
+  placeCallOptions = {videoOptions: {localVideoStreams:[localVideoStream]}};
+
+  localVideoView();
+  stopVideoButton.disabled = false;
+  startVideoButton.disabled = true;
+
+  const userToCall = calleeInput.value;
+  call = callAgent.startCall(
+    [{ communicationUserId: userToCall }],
+    placeCallOptions
+  );
+
+  call.remoteParticipants.forEach( p => {
+    subscribeToRemoteParticipant(p);
+  })
+
+  call.on('remoteParticipantsUpdated', e => {
+    e.added.forEach( p=>{
+      subscribeToRemoteParticipant(p);
+    })
+  });
```

Comment:
I see this repeating `call.remoteParticipants.forEach + call.on('remoteParticipantsUpdated'`
can we wrap this two in a function so we reuse it and just call something

`subscribeRemoteParticipantsInCall(call);`

which would do both: foreach + call.on